extensions/msal-node-extensions/src/broker/NativeBrokerPlugin.ts

```diff
@@ -646,12 +645,6 @@ export class NativeBrokerPlugin implements INativeBrokerPlugin {
             const { errorCode, errorStatus, errorContext, errorTag } =
                 error as MsalRuntimeError;
             switch (errorStatus) {
-                case ErrorStatus.InteractionRequired:
-                case ErrorStatus.AccountUnusable:
-                    return new InteractionRequiredAuthError(
```

Comment:
We can't remove this as this indicates to our customers that they should invoke an interactive API. If there's data missing let's see how we can include the needed info in the existing error object